index.js

```diff
@@ -31,13 +31,12 @@ async function run () {
     // Some setup required to make calls through WSO2
     await wso2.setOauthSettings(clientKey, clientSecret)
 
-    // Translate GitHub username into Net ID
-    const optionsToGetNetId = {
-      method: 'GET',
-      uri: `https://api.byu.edu:443/domains/servicenow/tableapi/v1/table/sys_user?sysparm_query=u_github_username=${githubUsername}&sysparm_fields=user_name`
-    }
-    const bodyWithNetId = await requestWithRetry(optionsToGetNetId)
-    const netId = bodyWithNetId.result[0].user_name
+    const netId = await getNetIdAssociatedWithGithubUsernameInServicenow(githubUsername).catch(() => {
+      error(`⚠ An error occurred while getting the Net ID associated with your GitHub username.
+Is your GitHub username associated with your profile in ServiceNow?
+You can check by going to https://support.byu.edu/nav_to.do?uri=%2Fsys_user.do%3Fsys_id%3Djavascript:gs.getUserID()%26sysparm_view%3Dess`)
```

Comment:
do we need to make any mention about ittest (or the sandbox of service now) here?

Reply:
Nope. When getting the GitHub username, it always checks production.